sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -233,12 +228,40 @@ protected ServiceBusClientBuilder getBuilder(boolean useCredentials) {
             .transportType(AmqpTransportType.AMQP)
             .scheduler(scheduler);
 
+        logger.info("Getting Builder using credentials : [{}] ", useCredentials);
         if (useCredentials) {
             final String fullyQualifiedDomainName = getFullyQualifiedDomainName();
 
             assumeTrue(fullyQualifiedDomainName != null && !fullyQualifiedDomainName.isEmpty(),
                 "AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME variable needs to be set when using credentials.");
+            String clientId = System.getenv("AZURE_CLIENT_ID");
+            String domainName = System.getenv("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME");
+            String clientSecret = System.getenv("AZURE_CLIENT_SECRET");
+            String tenantId = System.getenv("AZURE_TENANT_ID");
+
+            if (domainName != null ) {
+                logger.info("Getting Builder using credentials with domainName.length : [{}] ", domainName.length());
+            } else {
+                logger.error("Getting Builder using credentials domainName is null.");
```

Comment:
logging an error is not enough. the test should just fail with an assertNotNull

Reply:
I am removing it because  this was temporary , just for testing in pipeline.